bindings.xjb

```diff
@@ -1,24 +1,32 @@
 <jxb:bindings xmlns:jxb="https://jakarta.ee/xml/ns/jaxb"
-              xmlns:xs="http://www.w3.org/2001/XMLSchema"
-              xmlns:xsd="http://www.w3.org/2001/XMLSchema"
-              xmlns:xjc="http://java.sun.com/xml/ns/jaxb/xjc" jxb:version="3.0" jxb:extensionBindingPrefixes="xjc">
-    
-	<jxb:bindings>
-		<jxb:globalBindings underscoreBinding="asCharInWord">
-<!--			<xjc:javaType name="java.time.LocalDateTime" xmlType="xs:dateTime" adapter="org.rutebanken.util.LocalDateTimeISO8601XmlAdapter" /> -->
-			<xjc:javaType name="java.time.ZonedDateTime" xmlType="xs:dateTime" adapter="org.rutebanken.util.ZonedDateTimeISO8601XmlAdapter" />
-			<xjc:javaType name="java.time.LocalTime" xmlType="xs:time" adapter="org.rutebanken.util.LocalTimeISO8601XmlAdapter" />
-			<xjc:javaType name="java.time.LocalDateTime" xmlType="xs:date" adapter="org.rutebanken.util.LocalDateXmlAdapter" />
-			<xjc:javaType name="java.time.Duration" xmlType="xs:duration" adapter="org.rutebanken.util.DurationXmlAdapter" />
-        </jxb:globalBindings>
-	</jxb:bindings>
+  xmlns:xs="http://www.w3.org/2001/XMLSchema"
+  xmlns:xsd="http://www.w3.org/2001/XMLSchema"
+  xmlns:xjc="http://java.sun.com/xml/ns/jaxb/xjc"
+  jxb:version="3.0"
+  jxb:extensionBindingPrefixes="xjc">
 
-	<!-- See the script bin/version_updater.sh It updates version in schema locations-->
+  <jxb:bindings>
+    <jxb:globalBindings underscoreBinding="asCharInWord">
+      <xjc:javaType name="org.rutebanken.time.XmlDateTime" xmlType="xs:dateTime" adapter="org.rutebanken.util.XmlDateTimeAdapter" />
+      <!--<xjc:javaType name="java.time.ZonedDateTime" xmlType="xs:dateTime" adapter="org.rutebanken.util.ZonedDateTimeISO8601XmlAdapter" />-->
+      <xjc:javaType name="java.time.LocalTime" xmlType="xs:time" adapter="org.rutebanken.util.LocalTimeISO8601XmlAdapter" />
```

Comment:
I don't like that you have switched around this. If a user previously expected the Zoned variant, he now gets something else. That is not stable.

Reply:
Well, the XSD never made any guarantees about which one it is. Now you're confronted with that fact and have to adapt your code.

Reply:
Agreed, I already closed this thing.